src/main/resources/bin/launch_nn_loader

```diff
@@ -34,7 +34,7 @@ ${JAVA_HOME}/bin/java -server -XX:ParallelGCThreads=8 -XX:+UseParNewGC -XX:+UseC
 -Dcom.sun.management.jmxremote.authenticate=false \
 -Dnamenode.metrics.logger=INFO,NNMETRICSRFA -Dorg.mortbay.jetty.Request.maxFormContentSize=-1 \
 -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005 \
--cp "${JAVA_CLASSPATH}" com.paypal.namenode.NNAnalyticsRestAPI \
+-cp "${JAVA_CLASSPATH}" com.paypal.namenode.WebServerMain \
```

Comment:
Change is good. I think the name of this file should be `nna_launcher` however; given we are moving away from "NNLoader" naming/